.github/workflows/bundle.yaml

```diff
@@ -10,6 +10,9 @@ jobs:
     name: release
     runs-on: ubuntu-latest
     steps:
+    - name: Checkout
+      uses: actions/checkout@v2
```

Comment:
We ran into issues where the docker/build-push-action wasn't able to locate the Dockerfile.plainbundle, and adding the checkout step before running that job silenced that error.